swarm/network/stream/delivery.go

```diff
@@ -140,6 +140,17 @@ func (d *Delivery) handleRetrieveRequestMsg(sp *Peer, req *RetrieveRequestMsg) e
 	log.Trace("received request", "peer", sp.ID(), "hash", req.Key)
 	handleRetrieveRequestMsgCount.Inc(1)
 
+	// swap - record credit for 1 request
+	// note that only charge actual reqsearches
+	var err error
+	if sp.swap != nil {
+		err = sp.swap.Add(1)
+	}
+	if err != nil {
+		log.Warn(fmt.Sprintf("Delivery.handleRetrieveRequestMsg: %v - cannot process request: %v", req.Key.Log(), err))
```

Comment:
We have structured logs, no need to use `Sprintf`. I know this is done throughout the codebase, but we should stop doing it.